dagql/idtui/frontend_pretty.go

```diff
@@ -446,16 +626,58 @@ func (fe *frontendPretty) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			fe.restore()
 			sigquit()
 			return fe, nil
+		case "down", "j":
+			fe.goDown()
+			return fe, nil
+		case "up", "k":
+			fe.goUp()
+			return fe, nil
+		case "left", "h":
+			fe.goOut()
+			return fe, nil
+		case "right", "l":
+			fe.goIn()
+			return fe, nil
+		case "home":
+			fe.goStart()
+			return fe, nil
+		case "end", " ":
+			fe.goEnd()
+			return fe, nil
+		case "esc":
+			fe.zoomed = fe.db.PrimarySpan
+			fe.recalculateViewLocked()
+			return fe, nil
+		case "+":
+			fe.FrontendOpts.Verbosity++
+			fe.recalculateViewLocked()
+			return fe, nil
+		case "-":
+			fe.FrontendOpts.Verbosity--
+			if fe.FrontendOpts.Verbosity < 0 {
+				fe.FrontendOpts.Verbosity = 0
+			}
+			fe.recalculateViewLocked()
+			return fe, nil
+		case "c":
+			if fe.cloudURL != "" {
+				browser.OpenURL(fe.cloudURL)
```

Comment:
Can we give some visual feedback when this happens? I was testing this, didn't realize it had opened, hammered the button, and went to my browser later to see 100s of cloud tabs open?

Reply:
Oooh, maybe we could also make the `Cloud` text a hyperlink :eyes:

Reply:
Done! I think you might need to hold "Shift" for it to work though, not sure if worth it. Try it out in your terminal? This sort of thing varies a lot between implementations.